alhimick-prog/hometask-6/scrapper.rb

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'selenium-webdriver'
+require 'nokogiri'
+require 'capybara'
+require 'capybara/rspec'
+require 'capybara/dsl'
+require 'csv'
```

Comment:
@alhimick-prog do you really need this? If so, try to rewrite `File.open` to `CSV.open`